### ISSUE TITLE
Different colors for syntax highlighting in Goland and PyCharm for .go files

### ISSUE THREAD

**Issue by sugat009** [ChrisRM/material-theme-jetbrains#1786]
### Summary
I use Pycharm for python projects and Goland for go projects and I'm a huge fan of material themes, so I use it in both my IDEs. However, I noticed something is not right in Goland for the theme and the syntax highlighting is not "material" in nature. So I decided to open the go file in pycharm. The syntax highlighting is "material" in nature in pycharm for go files.

**Screenshot for goland with .go files**
![go_go_code](https://user-images.githubusercontent.com/30311933/123933406-15ea1300-d9b2-11eb-9ade-ea0f6dee10e5.png)

**Screenshot for pycharm with .go files**
![pycharm_go_code](https://user-images.githubusercontent.com/30311933/123933467-20a4a800-d9b2-11eb-9dac-9058793f192d.png)

### Expected behavior

The syntax highlighting should be "material" in nature in goland just like that of pycharm.

### Actual behavior

The syntax highlighting in goland is not "material" in nature.

**Goland info**

GoLand 2020.1
Build #GO-201.6668.125, built on April 9, 2020
Linux 5.10.42-1-manjaro
Registry: run.processes.with.pty=TRUE, ide.completion.variant.limit=500, ide.tree.collapse.recursively=false, suggest.all.run.configurations.from.context=TRUE, ide.balloon.shadow.size=0, ideFeaturesTrainer.welcomeScreen.tutorialsTree=TRUE
Non-Bundled Plugins: zielu.gittoolbox, com.arcticicestudio.nord.jetbrains, com.chrisrm.idea.MaterialThemeUI, com.github.redfoos.logstash-intellij-plugin, com.mallowigi, github.cweijan.test-theme, mobi.hsz.idea.gitignore, io.github.will7200.plugins.casbin, name.kropp.intellij.makefile, net.seesharpsoft.intellij.plugins.csv, org.jetbrains.plugins.hocon, izhangzhihao.rainbow.brackets, ru.adelf.idea.dotenv, idea.plugin.protoeditor, net.ashald.envfile
Current Desktop: KDE

Material Theme UI version: 5.3.2.1

_material_theme.xml_

`<application>
  <component name="MaterialThemeConfig">
    <option name="isWizardShown" value="true" />
    <option name="overrideAccentColor" value="false" />
    <option name="pristineConfig" value="false" />
    <option name="tabFontSizeEnabled" value="true" />
    <option name="tabFontSize" value="18" />
    <option name="accentColor" value="4285f4" />
    <option name="selectedTheme" value="DARKER" />
    <option name="version" value="5.2.0" />
  </component>
</application>`

_material_custom_theme.xml_
`<application>
  <component name="MaterialCustomThemeConfig">
    <option name="excludedColor" value="2e3c43ff" />
    <option name="accentColor" value="4285f4ff" />
    <option name="notificationsColor" value="323232ff" />
    <option name="treeSelectionColor" value="546e7a50" />
    <option name="highlightColor" value="425b67ff" />
    <option name="secondBorderColor" value="2a373eff" />
    <option name="tableSelectedColor" value="314549ff" />
    <option name="contrastColor" value="1e272cff" />
    <option name="disabledColor" value="415967ff" />
    <option name="secondaryBackgroundColor" value="32424aff" />
    <option name="buttonColor" value="2e3c43ff" />
    <option name="selectionForegroundColor" value="ffffffff" />
    <option name="selectionBackgroundColor" value="546e7aff" />
    <option name="textColor" value="607d8bff" />
    <option name="foregroundColor" value="b0bec5ff" />
    <option name="backgroundColor" value="263238ff" />
  </component>
</application>`

---

**Pycharm info**
PyCharm 2020.1 (Professional Edition)
Build #PY-201.6668.115, built on April 7, 2020
Linux 5.10.42-1-manjaro
Registry: ide.tree.collapse.recursively=false, ide.balloon.shadow.size=0
Non-Bundled Plugins: Batch Scripts Support, CMD Support, zielu.gittoolbox, com.alayouni.ansiHighlight, com.chrisrm.idea.MaterialThemeUI, com.github.redfoos.logstash-intellij-plugin, com.intellij.ideolog, com.mallowigi, ideanginx9, io.github.will7200.plugins.casbin, name.kropp.intellij.makefile, net.seesharpsoft.intellij.plugins.csv, com.perl5, org.jetbrains.plugins.hocon, izhangzhihao.rainbow.brackets, net.ashald.envfile, ru.adelf.idea.dotenv, ru.meanmail.plugin.requirements, org.sonarlint.idea, org.toml.lang
Current Desktop: KDE

Material Theme UI version: 5.3.2.1

_material_theme.xml_
`<application>
  <component name="MaterialThemeConfig">
    <option name="allowDataCollection" value="true" />
    <option name="isWizardShown" value="true" />
    <option name="overrideAccentColor" value="false" />
    <option name="pristineConfig" value="false" />
    <option name="tabFontSizeEnabled" value="true" />
    <option name="tabFontSize" value="18" />
    <option name="accentColor" value="4285f4" />
    <option name="selectedTheme" value="DARKER" />
    <option name="version" value="5.2.0" />
  </component>
</application>`

_material_custom_theme.xml_
`<application>
  <component name="MaterialCustomThemeConfig">
    <option name="excludedColor" value="2e3c43ff" />
    <option name="accentColor" value="4285f4ff" />
    <option name="notificationsColor" value="323232ff" />
    <option name="treeSelectionColor" value="546e7a50" />
    <option name="highlightColor" value="425b67ff" />
    <option name="secondBorderColor" value="2a373eff" />
    <option name="tableSelectedColor" value="314549ff" />
    <option name="contrastColor" value="1e272cff" />
    <option name="disabledColor" value="415967ff" />
    <option name="secondaryBackgroundColor" value="32424aff" />
    <option name="buttonColor" value="2e3c43ff" />
    <option name="selectionForegroundColor" value="ffffffff" />
    <option name="selectionBackgroundColor" value="546e7aff" />
    <option name="textColor" value="607d8bff" />
    <option name="foregroundColor" value="b0bec5ff" />
    <option name="backgroundColor" value="263238ff" />
  </component>
</application>`

**Comment by mallowigi** [comment]
Oh that well known "JetBrains updates broke the color schemes" issue... It's been a long time since the last one (3 years)?

I'll take a look.

**Comment by sugat009** [comment]
I figured out the issue in my case. I was trying to sync the keybindings of pycharm and goland so I accidentally exported all the settings of pycharm, not just the keybindings, which also included the color schemes of pycharm but the color schemes did not have the settings for go so when I imported the settings into goland, it changed the color scheme for go in material darker theme. So,  I reset the settings of goland and installed material theme again and it worked like a charm.
